app/views/users/sessions/new.html.erb

```diff
@@ -1,9 +1,9 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-  <% content_for :before_content, govuk_back_link(
-    text: "Back",
-    href: registrations_root_path)
-  %>
+    <% content_for :before_content, govuk_back_link(
+      text: "Back",
+      href: registrations_root_path)
+    %>
```

Comment:
I prefer this right at the top of the file, since it doesn't sit inside this div. (I know you didn't put it here, but you touched it last 😛 )